include/openenclave/bits/sgx/sgxtypes.h

```diff
@@ -1086,7 +1095,9 @@ typedef struct _sgx_key
 
 /* OE seal key default flag masks*/
 #define OE_SEALKEY_DEFAULT_FLAGSMASK (~SGX_FLAGS_NON_SECURITY_BITS)
-#define OE_SEALKEY_DEFAULT_MISCMASK (~SGX_MISC_NON_SECURITY_BITS)
+/* Include the bit 0, which represents the EXINFO option for SGX2 */
+#define OE_SEALKEY_DEFAULT_MISCMASK \
+    ((~SGX_MISC_NON_SECURITY_BITS) | SGX_MISC_FLAGS_PF_GP_EXIT_INFO)
```

Comment:
It's arguable whether enabling #PF/#GP info delivery in Enclave might have security implication. But if the feature did have security implication with regard to sealed data migration, the implementation should change `SGX_MISC_NON_SECURITY_BITS` definition,  instead of overriding the relationship between `OE_SEALKEY_DEFAULT_MISCMASK` and `SGX_MISC_NON_SECURITY_BITS`

More importantly, we need to consider the impact of changing `OE_SEALKEY_DEFAULT_MISCMASK`. If an old version of OE app built with legacy OE SDK sealed data, and an updated version of the App was built with new definition of OE_SEALKEY_DEFAULT_MISCMASK, the newer version of the App won't be able to unseal the sealed data of the previous version of the App (running on the same platform), as the flagmask and miscmask are all part of the SEAL key derivation `KEYDEPEDENCIES`., which affects the derived SEAL key,  according to Intel's Software Developer's Manual.

Reply:
Good point. Should we defer the changes to the OE_SEALKEY_DEFAULT_MISCMASK and have a separate discussion? I think it's important to maintain the backward compatibility while providing ways such that developers can derive the key based on MISCSELECT value.

Reply:
@mingweishih , yes, MISCSELECT implication to sealed data migration policy should be discussed separately. I prefer to keep `SGX_MISC_NON_SECURITY_BITS` and `OE_SEALKEY_DEFAULT_MISCMASK` definitions unchanged, for now.

Reply:
Reverted the change